.github/workflows/build-wheels-linux.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       smoke-test-script: ${{ matrix.smoke-test-script }}
       # Using "development" as trigger event so these binaries are not uploaded
       # to official channels yet
-      trigger-event: development
```

Comment:
I'm guessing this is the change that actually uploads the wheels from GHA rather than from CircleCI, right? In any case, could you please remove the comment above this line, since it is no longer valid?

Reply:
Ah yes - great point!